### DISCUSSION TITLE
can you add the function of automatically starting in full screen?

### DISCUSSION THREAD

**Discussion by wangxustage** [Marat-Tanalin/bsnes-mt#28] [Ideas]
Hello, can you add the function of automatically going full screen after startup? I need this feature, it's very convenient. Thank you very much.

**Comment by Marat-Tanalin** [comment]
Hello. bsnes, like higan, supports switching to full-screen mode right after start with the [`--fullscreen`](https://higan.readthedocs.io/en/stable/interface/higan-cli/) command-line option.

bsnes-mt supports this option too, and unlike bsnes, bsnes-mt allows to switch back to normal mode by pressing `Escape` with no need to prior define a custom hotkey for switching between full-screen and normal modes.
